modules/contact-form/grunion-contact-form.php

```diff
@@ -2038,13 +2038,36 @@ function process_submission() {
 		 * with this work around. */
 		add_filter( 'wp_insert_post_data', array( $plugin, 'insert_feedback_filter' ), 10, 2 );
 
+		$disabled_functions = explode( ',', ini_get( 'disable_functions' ) );
```

Comment:
Not the best way to do this.  Any item in `disable_functions` can be detected as disabled by testing it against `function_exists()` as per https://secure.php.net/manual/en/function.function-exists.php#67947

We won't be able to redefine it, but it's a tidier way to check.